renku/cli/__init__.py

```diff
@@ -173,6 +186,18 @@ def print_global_config_path(ctx, param, value):
 @click.pass_context
 def cli(ctx, path, renku_home, use_external_storage):
     """Check common Renku commands used in various situations."""
+    renku_path = Path(path) / renku_home
+
+    if not renku_path.exists() and not is_allowed_command(ctx):
+        click.echo(
+            WARNING +
+            '`{0}` is not a renku initialized repository.'.format(path)
```

Comment:
With this wording, you should write `renku-initialized` and `renku-managed` below. But I think it's ok to just say `renku repository`, e.g., 

```WARNING + '`{0}` is not a renku repository.'.format(path)```